log-viewer/modules/parsers/TreeParser.ts

```diff
@@ -1033,6 +1106,10 @@ class EventSericeSubBeginLine extends Method {
     this.text = `${parts[2]} ${parts[3]}`;
     this.group = this.type;
   }
+
+  getBreadcrumbText(): string {
```

Comment:
I might be being picky here but I feel like the treeparser and log line classes should not know about UI concepts.
I would get rid of this and convert the event name from the log line class to the breadcrumb label in the tree view code. 

Alternatively would introduce a generic concept of a label in the log line and use that in the breadcrumbs?

Reply:
I've been dithering on how much the classes should know, but this text varies a lot from line to line and so it seemed worth including. I don't think a generic label would be suitable as this text is specifically shortened to try and fit a decent breadcrumb trail on the screen (with additional detail in the tooltip).